mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -1572,10 +1754,20 @@ createCPUConvWithCPP(ModuleOp module, func::FuncOp &func,
     elemType = b.getI8Type();
   }
 
+  Value filter = block->getArgument(0);
```

Comment:
Similar to above, potentially make F32 buffers for parameters ahead of some action.